software/huly.yml

```diff
@@ -0,0 +1,27 @@
+name: "Huly"
+website_url: "https://huly.io"
+source_code_url: "https://github.com/hcengineering/platform"
+description: "All-in-One Replacement for Linear/Jira, Slack, Notion, Motion and more. Integrated with GitHub and others."
```

Comment:
Hi,

The description should explain what the software does.

You can use the following syntax to denote the relationship to Linea/Jira (https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines)

> If the project is presented as an alternative to another service or application, please mention it as (alternative to $PRODUCT1, $PRODUCT2) at the end of the description.

Reply:
Thank you, fixed!